source/_integrations/slack.markdown

```diff
@@ -117,7 +118,49 @@ Example for posting file from local path:
 
 Please note that `path` is validated against the `whitelist_external_dirs` in the `configuration.yaml`.
 
-Example for posting formatted attachment:
+Example for using the block framework:
+
+```json
+{
```

Comment:
We don't use JSON anymore, it isn't presented like that for the user anymore. Can we convert this to YAML?